documentation/Doxyfile

```diff
@@ -3,19 +3,18 @@
 # To create a new Doxyfile containing all available options, call `doxygen -g`
 
 # Get Project name and version from CMake
-PROJECT_NAME = @DOXYGEN_PROJECT_NAME@
-PROJECT_NUMBER = @DOXYGEN_PROJECT_VERSION@
+PROJECT_NAME = Greeter
```

Comment:
IMO it would be great if we can continue to pass CMake variables to Doxygen, as it would allow better version synchronisation among others. One way to achieve this without using StableCoder's CMake scripts would be to call `configure_file` from CMake. 

```
configure_file(
  ${CMAKE_CURRENT_LIST_DIR}/Doxyfile.in
  ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile
)
```

And set the `GenerateDocs` target's working directory to the `CMAKE_CURRENT_BINARY_DIR`.

Reply:
Nice. I was not sure how to do this exactly. Thanks. I made the change, it looks much better now.